server/mongodb/actions/Project.ts

```diff
@@ -46,6 +48,21 @@ export async function getChapterProject(
   return project;
 }
 
```

Comment:
See previous comments about endpoint access, which could apply to this file.

Reply:
I'm not sure what would apply to this file, which comments are you referring to?